### PR TITLE
fix: show rchash duration response in seconds

### PR DESCRIPTION
**Pull request by martinconic**
### Checklist

- [x] I have read the [coding guide](https://github.com/ethersphere/bee/blob/master/CODING.md).
- [ ] My change requires a documentation update, and I have done it.
- [ ] I have added tests to cover my changes.
- [x] I have filled out the description and linked the related issues.

### Description
Fixes [#4838](https://github.com/ethersphere/bee/issues/4838)

### CODE REVIEW

File: openapi/SwarmCommon.yaml [hunk 1]
```diff
@@ -1016,7 +1016,7 @@ components:
       type: object
       properties:
         duration:
-          type: integer
```
Comment (janos) [inline openapi/SwarmCommon.yaml:1019]: This is a some sort of a breaking change as previously integer parsing will fail on floating point number.

I am not sure how significant this is.

**Review by janos — APPROVED** [review]

File: openapi/SwarmCommon.yaml (hunk 1 above)
Comment (martinconic) [reply openapi/SwarmCommon.yaml:1019]: since `.Seconds()` returns float64 it was needed to have a `number` here instead of `integer`

Comment (istae) [reply openapi/SwarmCommon.yaml:1019]: maybe it's appropriate here to define a new "seconds" type with a nice description and example

Comment (martinconic) [reply openapi/SwarmCommon.yaml:1019]: seconds type added

**Review by istae — APPROVED** [review]

**Pull request merged**